sal/class.json-api-site-base.php

```diff
@@ -559,4 +579,31 @@ function is_domain_only() {
 	function get_blog_public() {
 		return (int) get_option( 'blog_public' );
 	}
+
+	function has_pending_automated_transfer() {
+		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
```

Comment:
hmmm, we should move this logic to `class.json-api-site-wpcom.php` instead of the `defined` check